tlsn/tlsn-core/src/transcript.rs

```diff
@@ -75,21 +80,25 @@ impl TranscriptSlice {
         Self { range, data }
     }
 
+    /// Gets a reference to range
     pub fn range(&self) -> &Range<u32> {
         &self.range
     }
 
+    /// Gets a reference to data
     pub fn data(&self) -> &Vec<u8> {
         &self.data
     }
 }
 
-#[derive(Serialize, Deserialize, Clone, PartialEq)]
+#[derive(Debug, Serialize, Deserialize, Clone, PartialEq)]
 /// A [Transcript] consists of a stream of bytes which were sent to the server
```

Comment:
We changed Transcript to contain bytes only in one direction, so let's change this comment to:

/// A [Transcript] contains either a stream of bytes which were sent to the server
/// or a stream of bytes which were received from the server. The Prover creates
/// separate commitments to bytes in each direction.